cmd/importer/db.go

```diff
@@ -209,6 +209,16 @@ func genColumnData(table *table, column *column) (string, error) {
 
 		data = append(data, '\'')
 		return string(data), nil
+	case mysql.TypeNewDecimal:
+		var data float64
+		var limit = int64(math.Pow10(tp.Flen))
+		var scale = math.Pow10(-tp.Decimal)
+		if isUnique {
+			data = float64(uniqInt64Value(column, -limit, limit)) * scale
+		} else {
+			data = float64(randInt64Value(column, -limit, limit)) * scale
+		}
+		return strconv.FormatFloat(data, 'f', -1, 64), nil
```

Comment:
Sometimes, we need to use NewDecimal in index, convert it to float makes equal decimal not match.